.circleci/config.yml

```diff
@@ -0,0 +1,100 @@
+version: 2
+jobs:
+  relay:
+    working_directory: ~/artsy/reaction
+    docker:
+    - image: circleci/node:8
+    steps: 
+      - checkout
+      - restore_cache:
+          keys:
+          - yarn-deps-{{ checksum "yarn.lock" }}
+          - yarn-deps-
```

Comment:
The cache keys listed here are important. Circle treats cache as an immutable store. Our strategy is to default to a cache key that's tied to the contents of the `yarn.lock` file. That means so long as the `yarn.lock` doesn't update, we pull down the same cache. If the `yarn.lock` is updated but isn't yet cached, it'll fall back and do a partial match against `yarn-deps-`... meaning it grabs the last good version. That'll give us at least a partial cache hit.

Reply:
Nice! I learned the partial cache hit here! 👍